src/views/BasicPage.vue

```diff
@@ -14,14 +14,51 @@
 				required: true
 			}
 		},
+		data() {
+			return {
+				paddingLeft: 0, 
+			}
+		},
 		components: {
 			'markdown-interpreter': MarkdownInterpreter,
+		},
+
+		created() {						
```

Comment:
excessive whitespaces
```suggestion
		created() {
```